src/views/textview/TextSegment.js

```diff
@@ -142,7 +142,7 @@ export function TextSegment({
       outputText = ChineseSegment(
         inputData,
         inputData.split('').map(TextSegmentChineseWord),
-        false
+        true
```

Comment:
With this change the boolean to select linebreak in the function is no longer needed at all so true/false can go as well as the corresponding value in the function `ChineseSegment`. I will take care.